lib/store/fulfillment/fulfillment.ex

```diff
@@ -69,25 +78,6 @@ defmodule Elementary.Store.Fulfillment do
     }
   end
 
-  defp calculate_taxes(%{rate: tax_rate} = tax, order) do
```

Comment:
Just using printful to calculate taxes now. At some point I would like to verify this against what Stripe says because it would be more useful to use Stripe here. But that's down the road.